sentry/const.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Versada <https://versada.eu/>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import collections
+import logging
+
+import odoo.loglevels
+
+_logger = logging.getLogger(__name__)
+try:
+    import raven
+    from raven.conf import defaults
+except ImportError:
+    _logger.debug('Cannot import "raven". Please make sure it is installed.')
+
+
+def split_multiple(string, delimiter=',', strip_chars=None):
+    '''Splits :param:`string` and strips :param:`strip_chars` from values.'''
+    if not string:
+        return []
+    return [v.strip(strip_chars) for v in string.split(delimiter)]
+
+
+SentryOption = collections.namedtuple(
+    'SentryOption', ['key', 'default', 'converter'])
+
+# Mapping of Odoo logging level -> Python stdlib logging library log level.
+LOG_LEVEL_MAP = dict([
+    (getattr(odoo.loglevels, 'LOG_%s' % x), getattr(logging, x))
+    for x in ('CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET')
+])
+DEFAULT_LOG_LEVEL = 'warn'
+
+DEFAULT_TRANSPORT = 'threaded'
+TRANSPORT_CLASS_MAP = {
+    'requests_synchronous': raven.transport.RequestsHTTPTransport,
+    'requests_threaded': raven.transport.ThreadedRequestsHTTPTransport,
+    'synchronous': raven.transport.HTTPTransport,
+    'threaded': raven.transport.ThreadedHTTPTransport,
+}
+
+ODOO_USER_EXCEPTIONS = [
+    'odoo.exceptions.AccessDenied',
+    'odoo.exceptions.AccessError',
+    'odoo.exceptions.MissingError',
+    'odoo.exceptions.RedirectWarning',
+    'odoo.exceptions.UserError',
+    'odoo.exceptions.ValidationError',
+    'odoo.exceptions.Warning',
+    'odoo.exceptions.except_orm',
```

Comment:
> Oh and `odoo.exceptions.QWebException`

Hm, `QWebException` does not look like a user error so maybe it would make sense to receive such errors?. Also, it's weird that `QWebException` is also defined in [addons/base/ir/ir_qweb/qweb.py](https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_qweb/qweb.py#L110) so I guess we should ignore both (if we decide to ignore).

> Take a look at the below though - maybe we could leverage something like this to make the module a bit more future-ready?

Do you mean that we should analize the names in `odoo.exceptions` and look for `Exception` subclasses? IDK, the list of ignored exceptions is configurable, so one could add any new exceptions to the list in the config file. Also, analizing the exceptions at load time would feel a bit _magic_, i.e. you wouldn't really know which exceptions are ignored. What do you think?